driller.py

```diff
@@ -0,0 +1,10 @@
+""" Uses Pydriller to extract information from Github."""
+from git import Repo
+from pydriller import RepositoryMining
+
+
+def find_repositories(repo):
+    """ Locates a Github repository with the URL provided by the user. """
+    # ask the user for a URL of a Github repository
+    miner = RepositoryMining(path_to_repo=repo)
+    return miner
```

Comment:
This should go into the `data_collection.py` since they are all for retrieving data using pyDriller from github. But I don't think this method is even necessary. It's just a pass-through method right?